packages/playground/windows/playground-win32/Application.manifest

```diff
@@ -18,6 +18,19 @@
     type="winb32"
     version="1.0.0.0" />
 
+  <dependency>
+    <dependentAssembly>
+      <assemblyIdentity
+          type="win32"
+          name="Microsoft.Windows.Common-Controls"
+          version="6.0.0.0"
```

Comment:
Why is this necessary?

Reply:
by default apps using comctl get comctl 5.82 version (XP style) which uses older controls.

Reply:
ok, so no risk of a win10 machine with a version < 6.0?

Reply:
correct, this should be available on Vista+